src/ci/scripts/should-skip-this.sh

```diff
@@ -18,7 +18,11 @@ BASE_COMMIT="$(git merge-base FETCH_HEAD HEAD)"
 
 echo "Searching for toolstate changes between $BASE_COMMIT and $(git rev-parse HEAD)"
 
-if git diff "$BASE_COMMIT" | grep --quiet "^index .* 160000"; then
+if git log HEAD..."$BASE_COMMIT" | grep --quiet "compiler/rustc_error_codes/src/error_codes/"; then
```

Comment:
Hm, how does this work?

This command fails over here even if no error codes have been touched. The `git log HEAD..."$BASE_COMMIT"` does not output a list of changed files or anything of the sort here. Furthermore, even if it did, I don't see how this checks for deletions specifically, as opposed to any changes to that directory.

Reply:
You're absolutely right, I badly pasted the command and didn't even pay attention... The real command is:

```
git log HEAD..."$BASE_COMMIT" | --diff-filter=D --summary | grep delete | grep --quiet "compiler/rustc_error_codes/src/error_codes/"
```

Reply:
Actually both syntaxes give the same result.